packages/rest/test/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -194,6 +194,44 @@ describe('RestServer.getApiSpec()', () => {
     });
   });
 
+  it('emits all media types for request body', () => {
+    const opSepc = anOperationSpec()
```

Comment:
Typo: `opSepc`, should be `opSpec`. Since this is the spec we are expecting to be created, and not the spec used to define the API, please rename the variable to `expectedOpSpec` or `expectedSpec` to make that distinction clear.